src/Microsoft.Identity.Client/UiEvent.cs

```diff
@@ -30,11 +30,13 @@ namespace Microsoft.Identity.Client
 {
     internal class UiEvent : EventBase
     {
+        public const string ConstUserCancelled = "user_cancelled";
```

Comment:
We don't need to fix it here. This will be fixed in the main logic PR, and then I'll rebase this one.